lib/mix/torch.ex

```diff
@@ -53,11 +53,11 @@ defmodule Mix.Torch do
 
   def inject_templates("phx.gen.html", format) do
     copy_from("priv/templates/#{format}/phx.gen.html", [
-      {"edit.html.eex", "priv/templates/phx.gen.html/edit.html.eex"},
-      {"form.html.eex", "priv/templates/phx.gen.html/form.html.eex"},
-      {"index.html.eex", "priv/templates/phx.gen.html/index.html.eex"},
-      {"new.html.eex", "priv/templates/phx.gen.html/new.html.eex"},
-      {"show.html.eex", "priv/templates/phx.gen.html/show.html.eex"},
```

Comment:
So, we looked into this already when the original slim PR was merged (#156).  The way the slime generator works, requires the templates to use the `eex` extension.  Can you point to where you are seeing this not to be the case and where using `slime` as the extension is what the slime generator expects?

Reply:
Sorry forgot to commit the other changes. In fact slime templates should have the slime extension, even in the Torch code. And `copy_from/2` can be change the extension as needed.

Reply:
In fact, it's true that it is not really a fix but a better naming. :)